### ISSUE TITLE
Outdated examples: can't draw text on an image

### ISSUE THREAD

**Issue by YassinEldeeb** [image-rs/imageproc#492]
Using this [example](https://github.com/image-rs/imageproc/blob/master/examples/font.rs) causes the following errors:

```sh
$ cargo run
   Compiling deadline v0.1.0 (C:\development\Rust\deadline)
error[E0432]: unresolved import `imageproc::drawing::text_size`
 --> src\main.rs:2:41
  |
2 | use imageproc::drawing::{draw_text_mut, text_size};
  |                                         ^^^^^^^^^ no `text_size` in `drawing`

error[E0277]: the trait bound `ImageBuffer<Rgb<u8>, Vec<u8>>: image::image::GenericImage` is not satisfied
  --> src\main.rs:28:5
   |
28 |     draw_text_mut(&mut image, Rgb([0u8, 0u8, 255u8]), 0, 0, scale, &font, text);
   |     ^^^^^^^^^^^^^ the trait `image::image::GenericImage` is not implemented for `ImageBuffer<Rgb<u8>, Vec<u8>>`
   |
   = note: required because of the requirements on the impl of `Canvas` for `ImageBuffer<Rgb<u8>, Vec<u8>>`

error[E0277]: the trait bound `Rgb<u8>: image::traits::Pixel` is not satisfied
  --> src\main.rs:28:5
   |
28 |     draw_text_mut(&mut image, Rgb([0u8, 0u8, 255u8]), 0, 0, scale, &font, text);
   |     ^^^^^^^^^^^^^ the trait `image::traits::Pixel` is not implemented for `Rgb<u8>`

Some errors have detailed explanations: E0277, E0432.
For more information about an error, try `rustc --explain E0277`.
error: could not compile `deadline` due to 3 previous errors

```

**Comment by YassinEldeeb** [comment]
@gregoa Thank you for mentioning a working version, It's a good solution to downgrade at least for now.

**Comment by liyongjing** [comment]
image = "0.24.0"
imageproc = { git = "https://github.com/image-rs/imageproc.git" }

**Comment by ascodelife** [comment]
> @gregoa Thank you for mentioning a working version, It's a good solution to downgrade at least for now.
after downgrading the version of image to  0.23.14, it doesn't work for me. i got the same error as you.
`image = "=0.23.14"`
`imageproc = "0.22"`

**Comment by ascodelife** [comment]
> @gregoa Thank you for mentioning a working version, It's a good solution to downgrade at least for now.

after downgrading the version of image to 0.23.14, it doesn't work for me. i got the same error as you.
image = "=0.23.14"
imageproc = "0.22"

**Comment by ascodelife** [comment]
> > @gregoa Thank you for mentioning a working version, It's a good solution to downgrade at least for now.
> 
> after downgrading the version of image to 0.23.14, it doesn't work for me. i got the same error as you. image = "=0.23.14" imageproc = "0.22"

it works as follow
>image = "0.24.1"
imageproc = { git = "https://github.com/image-rs/imageproc" }

**Comment by qqhann** [comment]
When specifying version `imageproc = "0.22.0"`, it seems `drawing/mod.rs` does not export `text_size`. It becomes:

```rs
mod text;
pub use self::text::{draw_text, draw_text_mut};
```

However, the example requires `text_size` and the latest master contains it.

Is `0.22.0` outdated? 
Is there a newer version or a pre-release version?

**Comment by qqhann** [comment]
> image = "0.24.0"
> imageproc = { git = "https://github.com/image-rs/imageproc.git" }

For now this works for me, as `0.22.0` seems to be outdated.

**Comment by foresterre** [comment]
Example which I verified still works in 0.24:

https://github.com/foresterre/sic/blob/74cb246ef71cc9e1fe53a37d9938f1b320a2517d/components/sic_image_engine/src/operations/draw_text.rs#L64-L72

**Comment by theotherphil** [comment]
This is fixed in version 0.23, which I've just published
